### PR TITLE
Add FXIOS-5722 [v112] SiteImageView logs

### PR DESCRIPTION
**Pull request by lmarceau**
## [FXIOS-5722](https://mozilla-hub.atlassian.net/browse/FXIOS-5722) https://github.com/mozilla-mobile/firefox-ios/issues/13178
Thought there would be more usecase I would find useful, but here's what I am suggesting to add logs into SiteImageView. Let me know if you think of other cases!

### CODE REVIEW

**Comment by lmarceau** [comment]
There's some issues buidling `Fennec` due to the import of `Logger` inside `SiteImageView`.

**Comment by isabelrios** [comment]
Closing to try to re-run on Bitrise

**Comment by lmarceau** [comment]
Build is green [here](https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/cdf99ee2-a7c1-46d9-a8dd-ca58f096df17)

**Comment by lmarceau** [comment]
For context on latest changes:
- The Logger package was removed and integrated in Common package instead. Nested dependencies weren't playing nice with us (Fennec couldn't resolve dependencies anymore) and we needed to simplify them.
- This change makes us able to achieve the goal to add logs into SiteImageView package.

File: Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import Foundation
-import Logger
+import Common
```
Comment (adudenamedruby) [inline Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift:6]: So, in some places I'm seeing this, where we remove Logger and add Common, and in others where we remove Logger and don't add Common. What the reasoning behind those?

**Review by adudenamedruby — APPROVED** [review]

File: Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift (hunk 1 above)
Comment (OrlaM) [reply Client/Frontend/Library/HistoryPanel/HistoryPanelViewModel.swift:6]: Some already had Common imported.

**Pull request merged**